hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/AbstractBasicMessageListener.java

```diff
@@ -145,6 +181,7 @@ public void run() {
             }
 
             getLog().tracef("Received basic message: %s", retVal.getBasicMessage());
```

Comment:
I realize this is the original code, but we really shouldn't be logging messages anymore - this gets back to if the message has sensitive information (like the authentication data) we shouldn't be putting that in the log file. So I would remove this tracef or perhaps just log the class name (not the message .toString)